tests/core/test_serialization.py

```diff
@@ -519,7 +519,107 @@ def test_checkpoint_config_print(
                     },
                 ],
             },
-            id="config_without_id",
+            id="config_without_any_ids",
+        ),
```

Comment:
Additional tests to cover all combinations of top-level `validation_id` and nested `id` (within `Checkpoint.validations` list)